3i-nodejs/recipes/setup-pm2.rb

```diff
@@ -6,3 +6,8 @@
   command 'env PATH=$PATH:/usr/bin pm2 startup ubuntu -u ubuntu --hp /home/ubuntu'
   user 'root'
 end
+
+execute "connect-pm2-to-keymetrics" do 
+  command 'pm2 link $KEYMETRICS_SECRET $KEYMETRICS_PUBLIC'
+  user 'root'
```

Comment:
When I ran this command manually, it was under the `ubuntu` user account rather than root.  It's not immediately obvious to me whether running this command as root sets things up right.  The safe thing to do may be to run it as `ubuntu`

Reply:
I submitted the review before noticing that we're trying to use env vars here.  I don't think that'll work.  The chef way to do this is to use "attributes" like what's used [here](https://github.com/thirdiron/chef-cookbooks/blob/master/livingroom_conf/recipes/configure_rabbitMQ.rb#L13).

These values get filled by specifying chef attributes for the opsworks layer as shown in the following screenshot:
![selection_058](https://user-images.githubusercontent.com/614939/37541058-39a104a0-2927-11e8-8cf4-e6e0965c8367.jpg)

Reply:
I thought there might be something like that - okay, will set them up there and send back once I got that configured. Thanks!